pkg/sql/catalog/lease/kv_writer.go

```diff
@@ -30,8 +30,9 @@ import (
 type kvWriter struct {
 	db *kv.DB
 
-	oldWriter bootstrap.KVWriter
-	newWriter bootstrap.KVWriter
+	preMRWriter   bootstrap.KVWriter
```

Comment:
some commentary on what `MR/mr` means here would be appreciated :) multi-region I suppose?